src/CoreBundle/private/library/classes/dbobjects/TCMSRecord.class.php

```diff
@@ -2337,4 +2337,33 @@ private function getRequestStateHash()
 
         return $stateHashProvider->getHash();
     }
+
+    public function getCellFormattingFunction(array $cellConfig, string $cellFormattingFunctionName = ''): ?array
+    {
+        $tdbClassName = get_class($this);
+
+        if ('' === $cellFormattingFunctionName && 'id' === $cellConfig['db_alias'] && 'ID' === $cellConfig['title']) {
+            $cellFormattingFunctionName = 'callBackUuid';
+        }
+
+        if (false === is_callable(array($tdbClassName, $cellFormattingFunctionName))) {
```

Comment:
Another early return which checks for cellFormattingFunctionName still being ''?

Reply:
added